### PR TITLE
filter out invalid option for BoosterDraft

### PR DESCRIPTION
**Pull request by kevlahnota**

### CODE REVIEW

File: forge-gui/src/main/java/forge/gamemodes/limited/LimitedPoolType.java [hunk 1]
```diff
@@ -5,16 +5,45 @@
...
         displayName = name;
     }
 
+    public static LimitedPoolType[] values(boolean draftable) {
+        if (!draftable) {
+            return values();
+        }
+
+        int n = 0;
+        for (LimitedPoolType lpt : values()) {
```
Comment (tool4ever) [inline forge-gui/src/main/java/forge/gamemodes/limited/LimitedPoolType.java:31]: this loop doesn't do anything?

Comment (kevlahnota) [reply forge-gui/src/main/java/forge/gamemodes/limited/LimitedPoolType.java:31]: @tehdiplomat can you check, i change this PR to reflect the one you suggested (removed the streams and filter).

Comment (tehdiplomat) [reply forge-gui/src/main/java/forge/gamemodes/limited/LimitedPoolType.java:31]: No he's right this loop is later down in line 38. This one is duplicate

Comment (tehdiplomat) [reply forge-gui/src/main/java/forge/gamemodes/limited/LimitedPoolType.java:31]: This loop is counting the size for the be est array

Comment (kevlahnota) [reply forge-gui/src/main/java/forge/gamemodes/limited/LimitedPoolType.java:31]: Ok so It counts all draftable then set the size of array and fill the array with valid enum values.

**Review by tool4ever — APPROVED** [review]

**Pull request merged**